opus/application/apps/help/templates/help/gettingstarted.html

```diff
@@ -0,0 +1,271 @@
+
+<p>As you enter search values, the number of results that are selected will
+    constantly update in the <b>Browse Results</b> tab. In addition,
+    you will be provided with the minimum/maximum values or date/times,
+    or the number of results for each multiple-choice item, based on your
+    current search criteria.</p>
+
+<p>Once you have selected entered some search constraints, it's time to browse
+    the results.</p>
+
+<h3>Browsing</h3>
+
+<p>The <b>Browse Results</b> tab shows the observations selected by your search
+    in one of two ways:</p>
+
+<ol>
+    <li>The <b>Gallery View</b> shows a grid of colored thumbnails. The meaning
+        of the thumbnail contents differs depending on the instrument, but in
+        general it attempts to capture the image including filter color or
+        wavelength range (for imaging cameras or spectrographs) or footprint
+        (for non-imaging spectrographic instruments).</li>
+    <li>The <b>Table View</b> shows a series of rows, one per observation,
+        containing the currently selected metadata fields.</li>
+</ol>
+
+<p>You can switch between the two views by clicking on <b>View Table</b> or
+    <b>View Gallery</b>.</p>
+
+<p>You can navigate through the result set using the scrollbar on the right,
+    which may be moved either up or down. As the scrollbar approaches one of its
+    extremes, additional data will be loaded, giving the appearance of a
+    continuous data set in either direction. You may also use the mouse wheel,
+    up/down arrow keys, or page up/down keys. To move a larger distance through
+    the result set, you may use the slider at the top of the screen. The
+    observation number you select will appear in the top-left thumbnail
+    (gallery) or top row (table).</p>
+
+<p>Each observation has a menu (<i class="fas fa-bars fa-xs"></i>) that allows
+    you to perform a variety of operations, including adding or removing the
+    observation from the cart, starting or ending a range of observations
+    to be added or removed from the cart, showing the details of the
+    observation, or downloading observation data. A single observation may also
+    be added to or removed from the cart by using Ctrl+Click, clicking on the
+    <i class="fas fa-cart-plus fa-xs"></i> or
+    <i class="far fa-trash-alt fa-xs"></i> icons in the gallery view,
+    or clicking on the checkbox in the table view. A range of observations may
+    also be added to or removed from the cart by using Shift+Click to start and
+    end the range. If the starting observation is not currently in the cart,
+    all observations in the range will be added; if the starting observation is
+    already in the cart, all observations in the range will be removed.
+    See the section on the <b>Cart</b> tab below for more information on
+    downloading.</p>
+
+<p>The sort order for all results may be modified by clicking on a column header
```

Comment:
do you want to note that the same sort order <and selected metadata> applies to both browse and cart?

Reply:
Done